glean_parser/parser.py

```diff
@@ -88,7 +89,8 @@ def _load_file(
 
     filetype = FILE_TYPES.get(schema_key)
 
-    for error in validate(content, filepath):
+    modified_content = util.remove_output_params(copy.deepcopy(content), "defined_in")
```

Comment:
Why does this need to be removed here?  This file *should* only affect things coming directly from an input file.  (I'm probably just not being creative enough about where this is called from: can you share where this breaks if you *don't* make this change?)

Reply:
@mdboom The content to be validated was loaded from ping and metric yaml files through [`load_yaml_or_json`](https://github.com/Iinh/glean_parser/blob/75dfc5e51925adf3bee4bdf763c626b29cb4024c/glean_parser/util.py#L149-L152), which uses `SafeLineLoader` that adds line number to the output. So to be able to successfully validate the content against the schema here, we need to remove the line number from the output first.